cmd/podman/containers/run.go

```diff
@@ -148,6 +148,21 @@ func run(cmd *cobra.Command, args []string) error {
 		imageName = name
 	}
 
+	// If this is a checkpoint image, invoke container restore.
+	checkpointImage, err := utils.IsCheckpointImage(registry.GetContext(), []string{imageName})
+	if checkpointImage {
+		if err != nil {
+			return err
+		}
+
+		var restoreOptions entities.RestoreOptions
+		_, err := registry.ContainerEngine().ContainerRestore(registry.GetContext(), []string{imageName}, restoreOptions)
```

Comment:
nit: could  you do this on one line

if _, err := registry ...

Reply:
Thank you Daniel, I have updated this to also print any error messages from the response.